online-inference/vllm/02-inference-service.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: vllm-inference-service
+  annotations:
+    networking.knative.dev/ingress-class: kourier.ingress.networking.knative.dev
+  labels:
+    knative.coreweave.cloud/ingress: kourier.ingress.networking.knative.dev
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/minScale: "1"
+        autoscaling.knative.dev/maxScale: "1"
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: gpu.nvidia.com/class
+                operator: In
+                values:
+                - A40
+              - key: topology.kubernetes.io/region
+                operator: In
+                values:
+                - LAS1
```

Comment:
May not want to enforce a specific region, or go with `ORD1`. Whatever the traditional approach is for these examples. People not in this region may find long load times.